docs/advanced-usage/access-control/private-docker-registries/index.md

```diff
@@ -0,0 +1,9 @@
+# Private Docker registries
+
+If your workflow needs to access private Docker images, follow the docs depending on the Docker registry.
+
+## CERN GitLab
+
+Go to your project's member tab (`https://gitlab.cern.ch/<you-username>/<your-repo-name>/-/project_members`) and add the [REANA user](https://gitlab.cern.ch/reana) as a collaborator of type "reporter" (to know more about project collaborators on GitLab visit the [official docs](https://docs.gitlab.com/ee/user/permissions.html#project-members-permissions)):
```

Comment:
`s/you-username/your-username/g` or just add the usual `johndoe` example persona.

Reply:
You can also make phrases shorter for bettter comprehension: 

... of type "reporter". This will ensure that REANA platform will be able to access your private repositories. In order to know more about GitLab project collaborators, please see _GitLab project membership permissions_ documentation.

Reply:
Thank you, much better now :)